graylog2-server/src/main/java/org/graylog2/dashboards/widgets/FieldChartWidget.java

```diff
@@ -65,15 +66,11 @@ public FieldChartWidget(MetricRegistry metricRegistry, Searches searches, String
         final ImmutableMap.Builder<String, Object> persistedConfig = ImmutableMap.<String, Object>builder()
```

Comment:
I'm not sure if using a `ImmutableMap.Builder` is a good idea here. It will fail on calling `build()` if there were any duplicate keys added to it.

Are duplicate keys possible in that context?